examples/vnet_private_endpoint/README.md

```diff
@@ -63,14 +63,15 @@ module "virtual_network" {
 # Create a Private DNS Zone for API Management
 module "private_dns_apim" {
   source  = "Azure/avm-res-network-privatednszone/azurerm"
-  version = "~> 0.2"
+  version = "0.4.0"
 
   domain_name = "privatelink.azure-api.net"
+  parent_id   = azurerm_resource_group.this.id
```

Comment:
curious about this parent_id, did resoutrce group name cause problems? There is nothing wrong with it, but perhaps it should be named resource_group_id?